app/assets/javascripts/webchat/README.md

```diff
@@ -0,0 +1,74 @@
+### GOVUK webchat
+
+Gov uk webchat has been made to make it easy to implement webchat across all GOV.UK pages.
+Departments within government have multiple webchat integrations, and we cannot support them all.
+
+To this end, we have created a unified API for the javascript implementation, and recommend that departments implement a proxy for their own availability api.
+
+
+#### availability api implementation
+
+we have [made a reference implementation](https://github.com/alphagov/reference-webchat-proxy) of a proxy for the egain api to illustrate how you would create one.
```

Comment:
there is nothing that needs documented in the open url? i have no idea what they should be doing once they have the endpoint,t hat part of the process is not for me to define.